data-tool/flows/common/extract_tracking_service.py

```diff
@@ -46,13 +46,14 @@ def reserve_for_flow(self, base_query: str, flow_run_id: str) -> int:
         init_query = f"""
         WITH candidate_corps AS ({base_query}),
         available_corps AS (
-            SELECT corp_num, corp_type_cd
+            SELECT corp_num, corp_type_cd, mig_batch_id
             FROM candidate_corps
             FOR UPDATE SKIP LOCKED
         )
         INSERT INTO {self.table_name} (
             corp_num,
             corp_type_cd,
+            mig_batch_id,
```

Comment:
Will you update the `colin_tracking` table in this work? It uses `batch_id` as placeholder and will raise error if you run the colin freeze flow.

Reply:
Get it, thanks!